acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/ThreadPantheonNodeRunner.java

```diff
@@ -104,6 +112,7 @@ public void startNode(final PantheonNode node) {
             .metricsSystem(noOpMetricsSystem)
             .metricsConfiguration(node.metricsConfiguration())
             .p2pEnabled(node.isP2pEnabled())
+            .graphQLRpcConfiguration(GraphQLRpcConfiguration.createDefault())
```

Comment:
Not sure why this was failing on my branch and not master - was getting a null pointer exception because this dependency wasn't set ...

Reply:
Any ideas @shemnon ??

Reply:
Looks like this was missed.  

The big issue I see is that the threaded acceptance runner is only ever run by developers, the process runner is run on the jenkins boxes instead and would have broke there.

Reply:
Ah, so maybe it was just a flaky test and when I ran it locally I got the null pointer?